client/src/app/page/create-update-user-page/create-update-side-bar-info/create-update-side-bar-info.component.html

```diff
@@ -36,37 +38,54 @@
       <div class="input-label">
         <label for="position">Position</label>
       </div>
-      <app-dropdown-info class="info-input-area" id="position" [pairList]='positions'
-                         (selected)="newUser.position = $event"
-                         [required]="true">
+      <app-dropdown-info class="info-input-area" id="position"
+                         [pairList]='positions'
+                         (selected)="user.position = $event"
+                         [required]="true"
+                         [update]="user.position"
+                         [positionIdentifier]="'position'">
       </app-dropdown-info>
     </div>
     <div class="info-form" *ngIf="ifChosenDevelopmentDepartment">
       <div class="input-label">
         <label for="teamLead">Teamlead</label>
       </div>
-      <app-dropdown-info class="info-input-area" id="teamLead" [pairList]='teamLeads'
-                         (selected)="newUser.teamlead = $event"
+      <app-dropdown-info class="info-input-area" id="teamLead"
+                         [pairList]='teamLeads'
+                         (selected)="user.teamlead = $event"
+                         [update]="user.teamlead?user.teamlead._id:user.teamlead"
```

Comment:
spaces

Reply:
I do not entirely understand how I should change this
Because this this how it is autoformatted in my code

Reply:
but why ```(selected)="user.teamlead = $event"``` with spaces?

Reply:
(selected)="user.teamlead=$event"  ?? without spaces?

Reply:
Correct format:
```html
[update]="user.teamlead ? user.teamlead._id : user.teamlead"
```

Reply:
all fixed